src/mps/projmpo.jl

```diff
@@ -62,8 +110,16 @@ function Base.eltype(pm::ProjMPO)
   return elT
 end
 
-(pm::ProjMPO)(v::ITensor) = product(pm,v)
+"""
+    size(pm::ProjMPO)
 
+The size of a ProjMPO is the dimension
+of the space on which it acts as a linear
+operator. Thus if a ProjMPO maps from a space
+with indices `(a,s1,s2,b)` to the space 
+`(a',s1',s2',b')` then the size 
+is `dim(a)*dim(s1)*dim(s1)*dim(b)`.
```

Comment:
Shouldn't this read that the size is `(d, d)` where `d = dim(a)*dim(s1)*dim(s1)*dim(b)`?

Reply:
Yes, I had missed that, thanks!